envs/default.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   - fiona=1.8.4
   - rasterio=1.0.25
   - rasterstats=0.13.0
-  - geopandas=0.4.1
+  - geopandas=0.8.2
```

Comment:
This is quite a drastic upgrade. Are we sure about the consequences?

What about this here: https://geopandas.readthedocs.io/en/latest/docs/user_guide/projections.html#upgrading-to-geopandas-0-7-with-pyproj-2-2-and-proj-6?

Reply:
I can't remember the exact reason for this upgrade, but I do remember that it was necessary to avoid some strange pyproj errors related to different packages depending on different PROJ versions. Let me check what happens...